en/streaming-integrator/docs/examples/handling-requests-with-errors.md

```diff
@@ -326,4 +352,104 @@ To manage the error in the Error Store Explorer, follow the procedure below:
     
     ```
     INFO {io.siddhi.core.stream.output.sink.LogSink} - ReceiveAndCount : TotalCountStream : Event{timestamp=1597857170244, data=[1], isExpired=false}    
-    ```
\ No newline at end of file
+    ```
+
+### Step 7: Test event failing at database level
+
+#### Step 7.1: Create a table
+
+Create database named `production` and select the production database for use. Use the following MySQL query to create 
+a table named `SweetProductionTable`
+
+    ```
+    CREATE TABLE SweetProductionTable (
+        batchID int NOT NULL,
+        amount double,
+        factoryID int,
+        PRIMARY KEY (batchID)
+    );
+    ```
+#### Step 7.2: Deploy the StoreRdbmsError siddhi application
+
+In this step, let's start the service at `http://localhost:8006/insertStream` via the `StoreRdbmsError` Siddhi 
+application
+
+In the **Siddhi Apps to Deploy** section, select the check box for the **StoreRdbmsError.siddhi** application. In the 
+**Servers** section, select the check box for the server you added. Then click **Deploy**.
+
+   ![Select StoreRdbmsError Siddhi Application and Server](../../images/handling-requests-with-errors/select-StoreRdbmsError-app-and-server.png)
+
+The following log is displayed in the Streaming Integrator console.
+    ```
+    INFO {org.wso2.carbon.streaming.integrator.core.internal.StreamProcessorService} - Siddhi App StoreRdbmsError deployed successfully
+    ```
+
+#### Step 7.2: Publish an event into StoreRdbmsError Siddhi application
+
+Send an event to the insertStream stream of the StoreRdbmsError Siddhi application by issuing the following CURL 
+command.
+
+    `curl --location --request POST 'http://localhost:8006/insertStream' --header 'Content-Type: application/json' --data-raw ' { "event": { "batchID": 1, "amount": 45.6, "factoryID": 102 } }'`
+
+Check the contents of `SweetProductionTable` to see the published event successfully stored in the table
+
+    ```
+    mysql> select * from SweetProductionTable;
+    +---------+--------+-----------+
+    | batchID | amount | factoryID |
+    +---------+--------+-----------+
+    |       1 |   45.6 |       102 |
+    +---------+--------+-----------+
+    1 row in set (0.00 sec)
+    ```
+
+Use the same curl command as above to publish another event. However, this time our event will fail since the batchID 
+column which is defined as the primary key has a duplicate value of 1 in the second message. You will see the following 
+log in the wso2si terminal.
```

Comment:
Shall we also mention the fact that, this will also be captured in the error store?

Reply:
Fixed in a04ccde